PicoProducer/config/config.json

```diff
@@ -13,4 +13,4 @@
   "nfilesperjob": 1, 
   "filelistdir": "samples/files/$ERA/$SAMPLE.txt", 
   "jobdir": "output/$ERA/$CHANNEL/$SAMPLE"
-}
\ No newline at end of file
+}
```

Comment:
Any way we can avoid this change? It does not look like it meaningfully changed the file. I've seen`git` do this before, but no idea how to solve it. Is it perhaps the file's metadata?

Reply:
I fixed the merge conflict in the text editor, where changing the endline is not easy to spot. I either way plan to revert this file to the master branch one before merging